### DISCUSSION TITLE
Building Blocks Mobi FcRn Modeling

### DISCUSSION THREAD

**Discussion by Salih-Benamara** [Open-Systems-Pharmacology/Forum#1191] [Questions & Problems]
Hello !!

1) First Question
Please i have an issue with this error, it appears whene i add the internalization reaction for Bevacizumab, and i think that the FcRn recycling is not complet at 100% in all tissues whene it happen, so the PK profile change so the elimination rate (even whene i delete the TMDD reaction )

2) Second Question 
PLease any one knows haw can i manage the reaction to obtain directly the parameters of synthesis Rate ( Ksyn) and to put a value for the Ksyn, instead of this reaction with the syn_fac 

Thank You !! 👍 👍 

![Erreur 1](https://user-images.githubusercontent.com/100346781/161923451-9d8a4280-2d39-4925-a05b-66d0102cfb5f.JPG)
![erreur 2 ](https://user-images.githubusercontent.com/100346781/161923455-8c86e8a5-dba0-4b61-9786-801f7e622a3e.jpg)
![erreur 3](https://user-images.githubusercontent.com/100346781/161923729-2484feee-bc50-40cb-8a0f-5c9bbbe7f735.JPG)

**Comment by StephanSchaller** [comment]
@Salih-Benamara 

1. you can ignore the warnings (but I am not sure I understood what you meant with "the FcRn recycling is not complet at 100% in all tissues whene it happen"
2. k_deg is used in the synthesis as well to make sure the steady-state concentrations are always the same. you can use the syn_fac to differentiate the synthesis from degradation and this changes the steady state of your target.

**Reply by Salih-Benamara** [reply]
Thank you for your anwser @StephanSchaller :D
